From 4f45c00d96bc1fc684cab3de7cd8d30301bbed5c Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Sat, 17 Aug 2019 11:11:10 +0800
Subject: [PATCH 2/2] arch:arm64:dts: change memory reservation into static
 method

Refer to sdk kernel commit 1d40ccc5723b("arm64: dts: r8a7795-h3ulcb:
Add reserved memory regions"). If there is no-map preproty in dts node.
remove alloc-ranges and size properties, and then add reg property so
that reserve memory statically.
In additional, refer to sdk kernel, change to memory reservation address
into 0x54000000.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
index c1a030dca4c1..90f236745a37 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
@@ -43,8 +43,7 @@
 		/* device specific region for Lossy Decompression */
 		lossy_decompress: linux,lossy_decompress@54000000 {
 			no-map;
-			alloc-ranges = <0x0 0x6d000000 0x0 0x03000000>;
-			size = <0x0 0x03000000>;
+			reg = <0x00000000 0x54000000 0x0 0x03000000>;
 		};
 
 		/* global autoconfigured region for contiguous allocations */
-- 
2.17.1

